src/Features/GrassLighting.cpp

```diff
@@ -52,52 +52,59 @@ void GrassLighting::DrawSettings()
 	}
 }
 
-void GrassLighting::ModifyGrass(const RE::BSShader*, const uint32_t descriptor)
+void GrassLighting::ProcessFrame(PerFrame* perFrameData, PerFrameVR* perFrameDataVR)
```

Comment:
I broke out a helper function since the data structure changes between VR and Flat. The code should be identical though although the diff doesn't really show it well.